src/main/java/org/folio/inventory/resources/EventHandlers.java

```diff
@@ -46,12 +47,16 @@
 
   private static final String DATA_IMPORT_EVENT_HANDLER_PATH = "/inventory/handlers/data-import";
   private static final String INSTANCES_EVENT_HANDLER_PATH = "/inventory/handlers/instances";
+  // value in ms
+  private static final int DEFAULT_HTTP_TIMEOUT = 3000;
 
   private final Storage storage;
 
-  public EventHandlers(final Storage storage, final HttpClient client) {
+  public EventHandlers(final Storage storage) {
     Vertx vertx = Vertx.vertx();
     this.storage = storage;
+    HttpClientOptions params = new HttpClientOptions().setConnectTimeout(DEFAULT_HTTP_TIMEOUT);
```

Comment:
What happens when the connection duration exceeds the connect timeout?

Reply:
Default TimeoutExeption. In mod-inventory, it is 5 seconds, but no need to wait for more than 3 for this case